### PR TITLE
Add hybrid content policy related questions to forms

### PR DESCRIPTION
**Pull request by kennylong**

### CODE REVIEW

File: request.md [hunk 1]
```diff
@@ -58,3 +58,7 @@
 9) Please provide a general description of how you will analyze this data.
 
 10) Where do you intend to share the results of your analysis?
+
+11) Is there a third-party tool that you are proposing to use for the collection of data (or measurement)? If so:
```
Comment (chutten) [inline request.md]:
```suggestion
11) Is there a third-party tool that you are proposing to use for this data collection? If so:
```

Comment (chutten) [reply request.md]: We may wish to highlight that Telemetry is not a third-party tool. A lot of people coming at this form are unfamiliar with our data collection mechanisms and so the definition of "third-party tool" may need to be spelled out.

...kinda makes me wish there was a Telemetry-specific variant of the form.

**Review by chutten — COMMENTED** [review]
Looks good to me, with maybe one or two small changes.

File: request.md (hunk 1 above)
Comment (kennylong) [reply request.md]: @chutten Would changing it to " a third-party tool (not Telemetry)" work?

Comment (chutten) [reply request.md]: If you put an i.e. in front of it that might be more future-proof. "(i.e. not Telemetry)" But yes

**Comment by first44y** [comment]
Looks good to me as well.

**Pull request merged**